packages/pipeline/src/scripts/update_radar_orderbook_usd_prices.ts

```diff
@@ -0,0 +1,83 @@
+import { logUtils } from '@0x/utils';
+import { Connection, ConnectionOptions, createConnection } from 'typeorm';
+
+import * as ormConfig from '../ormconfig';
+import { handleError, isCli } from '../utils';
+
+if (isCli()) {
```

Comment:
is this different from `if (require.main === module)`? If not, would prefer to do it [the standard way](https://nodejs.org/api/modules.html#modules_accessing_the_main_module).

Reply:
I got this from https://stackoverflow.com/questions/6398196/node-js-detect-if-called-through-require-or-directly-by-command-line

What made me decide to use this is a comment that implied it works in more situations, but down to change it for more clarity.